docker/.env.sample

```diff
@@ -0,0 +1,22 @@
+# Copy this file and name it `.env.`
+# See https://docs.octopize.io/docs/deploying/self-hosted/configuration
+
+ENV_NAME="company-prod"
+DB_HOST="db"
+
+# When serving locally (check entries in /etc/hosts):
+BASE_API_URL='http://api.company.local:8000'
+WEB_CLIENT_URL='http://www.company.local:3000'
```

Comment:
why `company`?

Reply:
This is for our potential clients to use too, which might not use our api, so I didn't want to put `api.octopize.local`, I wanted it to be generic.

Any suggestions